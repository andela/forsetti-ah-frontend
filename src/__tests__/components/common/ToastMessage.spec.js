import React from 'react';
import { shallow } from 'enzyme';
import ToastMessage from '../../../components/common/ToastMessage';

describe('Toast Message ', () => {
  it('should render ToastMessage correctly', () => {
      const wrapper = shallow(<ToastMessage />);

      expect(wrapper).toMatchSnapshot();
  });
});
