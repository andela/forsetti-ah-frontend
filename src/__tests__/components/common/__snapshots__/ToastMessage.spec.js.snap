// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Toast Message  should render ToastMessage correctly 1`] = `ShallowWrapper {}`;
